base/docs/basedocs.jl

```diff
@@ -1783,6 +1783,18 @@ Number
     Real <: Number
 
 Abstract supertype for all real numbers.
+
+```jldoctest
+julia> subtypes(Real)
+4-element Vector{Any}:
+ AbstractFloat
+ AbstractIrrational
+ Integer
+ Rational
```

Comment:
This fails on CI, not sure why.

Help for `subtypes` has a similar example, is this one being processed earlier in loading or something?